File: Practice/21/c++/BMI/BMI/BMI.cpp
#include <iostream>

using namespace std;

double BMI(double weight, double height) {
	return weight / height / height;
}

void printBMI(double BMI) {
	if(BMI < 18.5)  cout << "Underweight";
	if (18.5 <= BMI && BMI< 25.0)  cout << "Normal weight";
	if (25.0 <= BMI && BMI < 30.0)  cout << "Overweight";
	if (30.0 <= BMI)  cout << "Obesity";
}

int main() {
	setlocale(LC_ALL, "rus");
	double weight, height;
	cout << "Введите свой вес в килограммах и рост в метрах.(через пробел)\n";
	cin >> weight >> height;
	printBMI(BMI(weight, height));
}

File: Practice/21/py/BMI/BMI/BMI.py
def bmi(weight: float, height: float) -> float:
    return weight/height/height

def print_bmi(BMI: float) -> float:
    if(BMI < 18.5):  print("Underweight")
    if (18.5 <= BMI and BMI< 25.0):  print("Normal weight")
    if (25.0 <= BMI and BMI < 30.0):  print("Overweight")
    if (30.0 <= BMI):  print("Obesity")


weight,height = input("Введите свой вес в килограммах и рост в метрах.(через пробел)\n").split(" ")
print_bmi(bmi(float(weight),float(height)))